doc-content/shared-kie-docs/src/main/asciidoc/Install/testing-clustering-environment-proc.adoc

```diff
@@ -0,0 +1,122 @@
+[id='testing-clustering-environment-proc_{context}']
+= Testing that your infrastructure meets the minimum hardware and performance requirements for a viable high availability (HA) on-premises environment.
+
+When creating a production-ready high availability (HA) on-premises infrastructure for {CENTRAL}, you must ensure it meets the minimum hardware and performance requirements for a viable HA environment, and to mitigate performance related issues. This section describes the environment elements, an example environment, and how to test the performance.
+
+There are four main components; {CENTRAL}, the message system (AMQ), the indexing server ({DATAGRID}), and a shared file system (NFS/GlusterFS/Ceph).
+
+.Prerequisites
+
+* A network environment of at least 3 nodes is configured with the following layout:
++
+Node 1: {CENTRAL}
++
+Node 2: {CENTRAL}
++
+Node 3: {CENTRAL}, {DATAGRID}, and NFS
+
+.Procedure
+
+. To test the network speed, do the following tasks:
+.. In the command terminal of each server node, install `iPerf3`:
++
+----
+$ dnf install iperf3
+----
+.. In the command terminal of the NFS server node, start `iPerf3` in server mode:
++
+----
+$ iperf3 -s
```

Comment:
This should be done in server 3 because we are going to measure the network speed from S1 and S2 to S3

Reply:
Done, changed to "In the command terminal of the NFS server node (server node 3), start `iPerf3` in server mode:"